docs/resources/migration.md

```diff
@@ -0,0 +1,86 @@
+---
+id: migration
+title: Migration
+--- 
+
+## Migrating components from Bit v14- to Harmony
+
+Harmony and pre-Harmony components and scopes (aka collections) are mutually incompatible. So how do you migrate over your pre-Harmony bit components to Harmony and take advantage of all the great new features? 
+
+Here's how.
+
+## What's changed
+
+First let's go through the fundamental changes between legacy version of Bit and Harmony:
+
+**Component structure** - Harmony is opinionated when it comes to structuring components in the file-system. Now each component must be in its own directory, so you'll potentially have to move some files around to suit this new requirement.   
+**Dependencies** - Harmony doesn't allow relative import statements between components. Each Bit component always has a local, compiled module in node_modules - use them for all Bit component import statements in your code.  
+**Environments** - Harmony features a new approach for compilers/testers/etc called environments. Take the base environment per your component's stack (react, node, react-native) and customize it to fit any specific configuration requirement.  
+**Configuration** - In harmony config is managed in a new file format - `workspace.jsonc` and not in your `package.json`.  
+**Module names** - In harmony we removed the @bit prefix, and instead components are now pre-fixed by @<account_name>. Your import statements are likely to need updating.  
+
+
+### Moving from Legacy Bit to Harmony
+
+Now that we've outlined the above, the major steps required to migrate components over from using legacy versions of Bit (14-) to Harmony are as follows (details for each below).
+
+:::note Install Harmony via the new BVM tool
+Check out installation instructions [here](/getting-started/installing-bit), including simulataneously running both legacy and Harmony versions of bit on your computer 
+:::
+
+1. Reset your bit workspace to be a Harmony workspace - if you would like to maintain your legacy workspace as well (recommended), then you'll need to create a fresh clone of the project before resetting
+1. Copy over the component code, making sure all components files are contained under a single directory, including the entry file into your component. Then track the components with Bit, in much the same way you would with previous Bit versions.
```

Comment:
@benjgil , there is an easier way to achieve this step by running `bit migrate --harmony`. See this: https://github.com/teambit/bit/issues/2795 for more details.

It was written a while ago, and wasn't tested since then, so you can ignore it altogether. Up to you.